packages/adblocker/src/data-view.ts

```diff
@@ -429,8 +429,10 @@ export class StaticDataView {
     const byteLength = this.getLength();
     this.pos += byteLength;
 
-    // @ts-ignore
-    return String.fromCharCode.apply(null, this.buffer.subarray(this.pos - byteLength, this.pos));
+    return String.fromCharCode.apply(
+      null,
+      Array.from(this.buffer.subarray(this.pos - byteLength, this.pos)),
+    );
```

Comment:
I don't think this changes are necessary and actually helping. It's pretty clear that `Array.from` adds an additional loop on top. I'll put benchmark results here later.